package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "vite-component-librarie-template",
+  "name": "vite-component-librarie-template-release-version",
```

please return name to orginal

```suggestion
  "name": "vite-component-librarie-template",
```